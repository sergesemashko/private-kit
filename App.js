/**
 * Sample React Native App
 * https://github.com/facebook/react-native
 *
 * @format
 * @flow
 */

import React, { useEffect } from 'react';
import SplashScreen from 'react-native-splash-screen';
import { MenuProvider } from 'react-native-popup-menu';
import Entry from './app/Entry';

const App: () => React$Node = () => {
  useEffect(() => {
    SplashScreen.hide();
  }, []);

  return (
    <MenuProvider>
      <Entry />
    </MenuProvider>
  );
};

export default App;
