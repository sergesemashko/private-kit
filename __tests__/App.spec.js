/**
 * @format
 */

import 'react-native';
import React from 'react';
import App from '../App';
import {render} from '@testing-library/react-native';

jest.mock('../app/Entry', () => 'Entry');

it('renders correctly', () => {
  const {asJSON} = render(<App />);

  expect(asJSON()).toMatchSnapshot();
});
