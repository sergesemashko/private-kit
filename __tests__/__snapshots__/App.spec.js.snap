// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <MenuProvider>
    <Entry />
  </MenuProvider>
</View>
`;
